resources/js/main.js

```diff
@@ -0,0 +1,287 @@
+import ithenticateSimilarityScoreCell from "./Components/ithenticateSimilarityScoreCell.vue";
+
+pkp.registry.registerComponent("ithenticateSimilarityScoreCell", ithenticateSimilarityScoreCell);
+
+const {useLocalize } = pkp.modules.useLocalize;
+const { useApp } = pkp.modules.useApp;
+const { useNotify } = pkp.modules.useNotify;
+import { computed, watch } from "vue";
+import deduceFileStatus from "./fileStatus";
+
+function runPlagiarismAction(piniaContext, stageNamespace) {
+    
+    const { useUrl } = pkp.modules.useUrl;
+    const { useFetch } = pkp.modules.useFetch;
+    const { t } = useLocalize();
+
+    const { isOPS } = useApp();
+
+    const fileStore = piniaContext.store;
+    const { submission, submissionStageId } = fileStore.props;
+
+    const ithenticateQueryParams = computed(() => {
+        const fileIds = isOPS()
+            ? (fileStore?.galleys?.map((galley) => galley.file.id) || [])
+            : (fileStore?.files?.map((file) => file.id) || []);
+
+        return {
+            fileIds: fileIds,
+            submissionId: submission.id,
+            stageId: isOPS() ? pkp.const.WORKFLOW_STAGE_ID_PRODUCTION : submissionStageId
+        };
+    });
+
+    const { apiUrl } = useUrl(`submissions/${submission.id}/plagiarism/status`);
+    
+    const { 
+        fetch: fetchIthenticateStatus, 
+        data: ithenticateStatus,
+    } = useFetch(
+        apiUrl, { 
+            method: 'POST',
+            body: ithenticateQueryParams 
+        }
+    );
+    
+    watch(ithenticateQueryParams, (newQueryParams) => {
+        if (newQueryParams?.fileIds?.length) {
+            fetchIthenticateStatus();
+        }
+    });
+
+    if (isOPS()) {
+        fetchIthenticateStatus();
+    }
+
+    const { pageUrl } = useUrl(`notification/fetchNotification`);
+    const { fetch: fetchNotification, data: notifications } = useFetch(pageUrl);
```

Comment:
Is fetching notifications still needed? I think we discussed just to leave it on frontend to present notifcation based on the API response?

Reply:
Yes, not using it anymore . just forgot to clean it up .